docs/backups.md

```diff
@@ -1,25 +1,67 @@
 # Providing Backups
 
-The Operator usually stores Percona XtraDB Cluster backups outside the
-Kubernetes cluster, on [Amazon S3 or S3-compatible storage :octicons-link-external-16:](https://en.wikipedia.org/wiki/Amazon_S3#S3_API_and_competing_services),
-or on [Azure Blob Storage :octicons-link-external-16:](https://azure.microsoft.com/en-us/services/storage/blobs/):
+It's important to back up your database to keep your data safe. 
+Backups help protect your system against data loss and corruption and ensure business stability. They are also a quick way to recover the database if something happens with it.
+
+A backup starts after you create a Backup object. You can create a Backup object in two ways:
+
+* manually at any moment. This way you start an [on-demand backup](backups-ondemand.md).
+* instruct the Operator to create it automatically according to a schedule that you define for it. This is a [scheduled backup](backups-scheduled.md).
+
+The Operator does physical backups using the [Percona XtraBackup :octicons-link-external-16:](https://docs.percona.com/percona-xtrabackup/2.4/index.html) tool and the [SST :octicons-link-external-16:](https://galeracluster.com/library/documentation/sst.html) method. 
```

Comment:
@nastena1606 Why do we have pxb 2.4 but not https://docs.percona.com/percona-xtrabackup/8.0/. The 2.4 is used only for PXC 5.7

Reply:
Fixed